frontend/pulse3d/src/components/uploadForm/AnalysisParamForm.js

```diff
@@ -333,6 +343,39 @@ export default function AnalysisParamForm({
             />
           </DropDownContainer>
         </ParamContainer>
+
+        <ParamContainer style={{ width: "33%", marginTop: "2%" }}>
```

Comment:
This should only be rendered if the selected pulse3d version supports this param

Reply:
Should I use a similar JS library for checking versions?